src/metrics/util.rs

```diff
@@ -74,19 +75,12 @@ pub(super) async fn count_pull_requests(
     graphql: &mut Graphql,
     org_name: &str,
     repo_name: &str,
-    time_period: Duration,
+    start_date: &Datetime,
+    end_date: &Datetime,
 ) -> usize {
-    // get date string to match GitHub's PR query format for `created` field
-    // i.e., "2021-05-18UTC" turns into "2021-05-18"
-    let date_str = chrono::NaiveDate::parse_from_str(
-        &format!("{}", (Utc::now() - time_period).date())[..],
-        "%FUTC",
-    )
-    .unwrap();
-
     let query_string = format!(
-        r#"repo:{}/{} is:pr created:>{}"#,
-        org_name, repo_name, date_str,
+        r#"repo:{}/{} is:pr created:{}..{}"#,
```

Comment:
Hmm, I wonder if we should change the toml file to just take this date string? Still, I can't imagine wanting anything other than a date range.

Reply:
As in... 

accept this: 
```
[data_source]
date = "2021-07-01..2021-08-01"
```

instead of this:

```
[data_source]
start_date = 2021-07-01
end_date = 2021-08-01
```

Reply:
That's what I meant, but I think I like this form better, it feels easier to remember and more obvious.

Reply:
Okay, cool :smile:  — I think I also like this way better.

Also I love that `.toml` has first-class dates, so no string! I have not noticed that previously.

And if it's an invalid date-time, the `toml::value::Datetime` will fail to parse before we ever send it for GitHub's GraphQL to reject — no extra work from us!